src/actions/preview.js

```diff
@@ -28,7 +28,11 @@ import type { AstLocation } from "../workers/parser";
 
 async function getReactProps(evaluate) {
   const reactDisplayName = await evaluate(
-    "this._reactInternalInstance.getName()"
+    "if (this._reactInternalFiber) {\n" +
+      "this._reactInternalFiber.type.name;\n" +
+      "} else {\n" +
+      "this._reactInternalInstance.getName();" +
+      "}"
```

Comment:
`this.hasOwnProperty("_reactInternalFiber") ? this._reactInternalFiber.type.name : this._reactInternalInstance.getName()`

Reply:
Haha, I'll make this change, and I just saw a test is failing, I will look at that too right away.